kotlin/internal/jvm/compile.bzl

```diff
@@ -266,14 +266,13 @@ def _fold_jars_action(ctx, rule_kind, toolchains, output_jar, input_jars, action
 def _resourcejar_args_action(ctx):
     res_cmd = []
     for f in ctx.files.resources:
-        c_dir, res_path = _adjust_resources_path(f.short_path, ctx.attr.resource_strip_prefix)
+        res_path = _adjust_resources_path(f.short_path, ctx.attr.resource_strip_prefix)
         target_path = res_path
         if target_path[0] == "/":
             target_path = target_path[1:]
-        line = "{target_path}={c_dir}{res_path}\n".format(
-            res_path = res_path,
+        line = "{target_path}={f_path}\n".format(
             target_path = target_path,
-            c_dir = c_dir,
```

Comment:
This was the only usage of `c_dir`. Since we don't need it anymore I removed it from the return values as well.